src/playground/service.clj

```diff
@@ -13,16 +13,27 @@
    [playground.services.invoices.insert.endpoint :as invoices.insert]
    [playground.services.invoices.retrieve.endpoint :as invoices.retrieve]
    [playground.services.invoices.delete.endpoint :as invoices.delete]
-   [playground.services.invoices.retrieve-all.endpoint :as invoices.retrieve-all]))
+   [playground.services.invoices.retrieve-all.endpoint :as invoices.retrieve-all]
+   [playground.views :as views]
+   ))
 
-(defn about-page [request]
+#_(defn about-page [request]
```

Comment:
Prefer deleting code to commenting it out. You always can revert to an older version, which is part of the point of Git